src/wren_core.c

```diff
@@ -878,6 +878,24 @@ DEF_PRIMITIVE(num_cos)
   RETURN_NUM(cos(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_decimal)
```

Comment:
This isn't the right name, since it doesn't deal with decimal representation. How about `fraction`?